testsuite/openshift/objects/auth_config/sections.py

```diff
@@ -175,20 +177,59 @@ def uma_metadata(self, name, endpoint, credentials, **common_features):
 class Responses(Section):
     """Section which contains response configuration"""
 
-    def add_simple(self, auth_json, name="simple", key="data", **common_features):
-        """Adds simple response to AuthConfig"""
-        self.add(
+    def _add(
+        self,
+        name: str,
+        value: dict,
+        wrapper_key: str = None,
+        wrapper: Literal["httpHeader", "envoyDynamicMetadata"] = None,
+        **common_features
+    ):
+        """Add response to AuthConfig"""
+        value["wrapper"] = wrapper
+        value["wrapperKey"] = wrapper_key
+
+        self.add_item(name, value, **common_features)
+
+    def add_simple(self, auth_json: str, name="simple", key="data", **common_features):
+        """Add simple response to AuthConfig"""
+        self.json(name, [Property(key, ValueFrom(auth_json))], **common_features)
+
+    @modify
+    def json(self, name: str, properties: list[Property], **common_features):
```

Comment:
My recommendation is to use `add_` in the names of these new functions. I see that this proposed form follows existing convention, in my opinion this short prefix has clear benefits. This is for discussion and broader agreement though (I am sorry if that already happened)

Reply:
I believe we decided to change the names of these functions in another PR. You are right that for now it just follows an existing convention